core/src/main/scala/org/bitcoins/core/script/ScriptProgram.scala

```diff
@@ -108,6 +108,19 @@ case class ExecutionInProgressScriptProgram(
   override def failExecution(error: ScriptError): ExecutedScriptProgram = {
     ScriptProgram.toExecutedProgram(this).failExecution(error)
   }
+
+  def addFlags(newFlags: Seq[ScriptFlag]): ExecutionInProgressScriptProgram = {
+    this.copy(flags = newFlags)
```

Comment:
If you want to keep this, you need to do 

```
this.copy(flags = (flags ++ newFlags).distinct)
```

Reply:
I meant to remove it, I think my rebase might have added it back on accident

Reply:
This adds the new flags to our existing flags, and then makes sure we do not have duplicates.

Reply:
ok, go ahead and remove and ignore comments. Rebasing can be tricky